### DISCUSSION TITLE
why dosen't it use my GPU

### DISCUSSION THREAD

**Discussion by 15xa** [lllyasviel/Fooocus#2919] [Q&A] [answered]
dosent Seem to use my GPU or CPU. And the usage of the ram is 100 percent. And also the storage. But the storage is an mva sshd. But still, it is being used 100. And the gpu was just being used 10%-30% and below. and 40-60% of the cpu on average, i don't understand what to do, each image takes 60 minutes, which is tooo slow, i gotta wait hours. not a great gpu, but there no usage is my concern. 
gpu- nv 1650, cpu ry 4600H
![image](https://github.com/lllyasviel/Fooocus/assets/62839670/151f5b22-245d-49e9-889f-75c9f82fe1fb)
![image](https://github.com/lllyasviel/Fooocus/assets/62839670/f3560437-e551-4b22-9d33-747ec4b1f34d)

**Comment by mashb1t** [comment, marked as answer]
This is expected, you're on bare minimum specs and bottlenecked by VRAM, which is why RAM is also fully used as fallback and disk too as 3rd fallback. The model basically is sharded over VRAM, RAM and swap (disk), which makes every sample step very slow. If everyhing would be loaded in VRAM it would be much faster.
Also see https://github.com/lllyasviel/Fooocus/issues/2920#issuecomment-2110830817
